### DISCUSSION TITLE
`target="_blank"` option for Link button in textarea field

### DISCUSSION THREAD

**Discussion by PaulBaileyAtChan** [ExpressionEngine/ExpressionEngine#3090] [Feature Requests]
It's a small thing, but when adding a link to content in the Entry Manager, using the chain/link dialogue box, it would be nice to have the option to mark the link to open in a new tab/window, and have this automatically rendered with the `target="_blank"` attribute. I find myself typing this in by hand _a lot_, and for non-techies a simple checkbox in the dialogue box, marked with something like "Open link in new tab/window", would make the feature much more user-friendly and accessible.

**Comment by intoeetive** [comment]
@PaulBaileyAtChan do you refer to links in RTE field, or something else?

**Comment by PaulBaileyAtChan** [comment]
Sorry, I wasn't clear. I was specifically referring to the process of adding a link to a textarea field using the HTML formatting button for a hyperlink. I have no idea if a similar situation would apply in an RTE field.
